mccodec/mc/play/world/clientbound_update_advancements.mbt

```diff
@@ -0,0 +1,13 @@
+///|
+/// Update Advancements packet (0x7B)
+/// Updates advancement data
+struct UpdateAdvancementsS2C {
```

Comment:
show advancements is missing

https://minecraft.wiki/w/Java_Edition_protocol/Packets#Update_Advancements